src/ATen/Context.h

```diff
@@ -13,6 +13,7 @@ namespace at {
 
 class ATen_CLASS  Context {
 public:
+
```

Comment:
Can you revert these whitespace changes?

Reply:
yep, certainly, I was finding it hard to read through the codebase and added these spaces after certain function ends.